src/cards.ts

```diff
@@ -19,11 +21,33 @@ const sets = CardInfo.reduce(
     {} as Dictionary<Expansion>
 );
 
-const cards: Card[] = CardInfo.map(card => ({
+const validCards = CardInfo.filter(card => (
```

Comment:
Try doing the `filter()` on the `cards` array (that has already gone through `map()`) once.